src/component/App/App.js

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
 import ButtonPanel from '../buttonPanel';
 import Display from '../Display';
+import calculate from '../../logic/calculate';
 
-function App() {
-  return (
-    <>
-      <Display />
-      <ButtonPanel />
-    </>
-  );
-}
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '',
+      next: '',
+      operation: '',  // eslint-disable-line
+    };
+  }
+
+  handleClick = (buttonName) => {
+    this.setState((prevState) => calculate(prevState, buttonName));
+  };
 
+  render() {
+    const { total, next } = this.state;
+    return (
+      <>
```

Comment:
- [ ] Please, always remember to make your commits in the present imperative tenses. You have committed your codes in past tense. 
- [ ] Please, do not disable eslint as you did here on line 12. There's a simple work around for this eslint error. Please, let's fix it. :pray: 
- [ ] I got this error `Error: [big.js] Division by zero` when i tried to divide a number by zero. Please, let's handle this exception.